src/julia.h

```diff
@@ -157,6 +157,7 @@ typedef struct {
     unsigned short how:2;
     unsigned short isshared:1;  // data is shared by multiple Arrays
     unsigned short isaligned:1; // data allocated with memalign
+    unsigned short fixed_len:1; // data cannot be resized (1d only)
```

Comment:
Sure. I did this quickly mostly to see how much of Base would break. Turns out not too much. I'm not sure how much people would miss dimensions 512-1023 but no reason to waste bits.